docs/framework-support-libraries.md

```diff
@@ -43,6 +43,7 @@ imports: [
 These are the main features provided by the library:
 * [LuigiContextService](#LuigiContextService)
 * [LuigiAutoRoutingService](#LuigiAutoRoutingService)
+* [Preload component](##LuigiAutoRoutingService)
```

Comment:
no explanation, link not working
Please add something like
"An empty angular component that can be used to build a preload route, see [link to viewgroup...preloadurl]"

Reply:
Updated in e01d9f7